android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java

```diff
@@ -27,14 +27,14 @@ public final class WalletConstants {
     public static List<String> BUY_SUPPORTED_NETWORKS = Arrays.asList(
             BraveWalletConstants.MAINNET_CHAIN_ID, BraveWalletConstants.RINKEBY_CHAIN_ID,
             BraveWalletConstants.ROPSTEN_CHAIN_ID, BraveWalletConstants.GOERLI_CHAIN_ID,
-            BraveWalletConstants.KOVAN_CHAIN_ID, BraveWalletConstants.LOCALHOST_CHAIN_ID,
-            BraveWalletConstants.POLYGON_MAINNET_CHAIN_ID,
-            BraveWalletConstants.BINANCE_SMART_CHAIN_MAINNET_CHAIN_ID,
-            BraveWalletConstants.AVALANCHE_MAINNET_CHAIN_ID,
-            BraveWalletConstants.CELO_MAINNET_CHAIN_ID,
-            BraveWalletConstants.FANTOM_MAINNET_CHAIN_ID,
-            BraveWalletConstants.OPTIMISM_MAINNET_CHAIN_ID,
-            BraveWalletConstants.AURORA_MAINNET_CHAIN_ID);
+            BraveWalletConstants.KOVAN_CHAIN_ID, BraveWalletConstants.LOCALHOST_CHAIN_ID);
```

Comment:
shouldn't all except `MAINNET_CHAIN_ID` be commented to hide the buy option on `buy-send-swap` bottom sheet as well, to indicate that we don't support buy for test networks at the first place

Reply:
It's for desktop parity, they have a message for test nets and disables `Buy` tab for the rest. Until they change that we will do this too.

Reply:
I see it's not the case anymore from the picture you posted in the PR. Doesn't desktop forward you to a faucet for Ropsten?

Reply:
@SergeyZhukovsky no they show the message in the Buy tab, I think I sent a screenshot yesterday?

Reply:
ok, sounds good!

Reply:
Not sure about desktop but still it doesn’t make sense to show buy option when we just gonna show no support for buy on the next screen. Good to be confirmed with @jamesmudgett

Reply:
BTW Desktop doesn't have two steps to show buy screen, everything is side by side, only mobile has the option of `buy-send-swap` chooser dialog so it's specific to mobile.

Reply:
Talked to @jamesmudgett yesterday, he is good to not show buy option for unsupported networks, like Solana.

Reply:
I have update to disable Buy on test nets too.